frontend/src/DeviceDetailsPage/DeviceDetailsPage.jsx

```diff
@@ -25,25 +26,27 @@ export default function DeviceDetailsPage(props) {
           [device.name, device.id]
         ]}
       />
-      <Grid container spacing={1}>
-        <Grid item xs={12} className="flexContents headerAreaUnderline">
-          <div className="title">{device.name}</div>
-        </Grid>
-        <Grid item xs={12} />
-        <Grid container xs={5} spacing={2}>
-          <Grid item xs={12}>
-            <DeviceDetailsConciseTable device={device} />
-          </Grid>
-          <Grid item xs={2}>
-            <Button variant="contained" color="primary">
-              Edit
-            </Button>
-          </Grid>
-          <Grid item xs={2}>
-            <Button variant="outlined">Delete</Button>
-          </Grid>
+      <Box className="areaUnderBreadcrumbs">
+        <Box className="flexContents headerAreaUnderline">
+          <Box className="flexContents">
+            <div className="title">{device.name}</div>
+            <Box padding={4} paddingLeft={1} paddingBottom={0}>
+              <Button>
+                <EditIcon color="action" />
+              </Button>
+            </Box>
+          </Box>
+          <div className="alignRightFloat">
+            <Box marginRight={2} marginTop={2}>
+              <DeleteDeviceButton deleteId={device.serialNumber} />
+            </Box>
+          </div>
+        </Box>
+      </Box>
+      <Grid container>
+        <Grid item xs={6}>
+          <DeviceDetailsTabTable tabs={["Overview"]} device={device} />
```

Comment:
I don't think you need to have the tab here. since this doesn't use the tab functionality. I reckon you can just leave it as a `DeviceDetailsConciseTable` component only.

Reply:
Indeed there is nothing to "tab through", however I'm using the Tab component for aesthetic/consistency reasons to go alongside the TabPanels on the right hand side.

Reply:
![image](https://user-images.githubusercontent.com/33813045/105560576-251db400-5ce2-11eb-8b7d-3f6e9cf4d053.png)